episodes/introduction.md

```diff
@@ -30,17 +30,59 @@ Objectives
 
 ## Introduction
 
-Placeholder text. 
+Placeholder text.
+
+:::{image} ../media/episodes/introduction/hub-overview.png
+:width: 100%
+:alt: Diagram showing an overview of a 2i2c Hub, includiing authentication, online content, custom environments, the community hub, cloud infrastructure, support and services.
+
+:::
 
 ## Exercises
 
-::::{dropdown} Challenge Title
+::::{dropdown} Which of the following statements are True/False?
 :open:
 
-Challenge content
+a. The infrastructure for running a 2i2c-managed hub is closed source and not replicable across different cloud providers.  
+b. Access to the hub can be controlled with authentication and authorisation.  
+c. Software environments are inflexible and irreproducible.  
+d. 2i2c provides operations support for your hub.
 
 :::{dropdown} Solution
-Solution content
+a. False – Hubs are built entirely with open source and community-driven tooling. The Right to Replicate gives communities the right to replicate their infrastructure in its entirety elsewhere.  
+b. True – Hub Champions can control who can access the hub through GitHub and other authentication methods.  
+c. False – Software environments are managed by Hub Champions and automatically deployed with containerisation, allowing for scalability across the cloud and reproducibile user environments.  
+d. True – 2i2c engineers maintain service availability, uptime and operational upgrades. Hub Champions are responsible for hub configuration and management to support their user community.
+
+:::
+
+::::
+
+:::{dropdown} What are the needs of your user community?
+:open:
+
+In pairs, discuss and share the following points:
+
+- What domain of expertise does your community have?
+- How large is your community? Do they work alone or do they need to collaborate together?
+- Does your community work with large datasets?
+- Does your community need access to intense computational power?
+- What are the main software applications your community uses to conduct their work? 
+- How familiar is your user community with version control using git/GitHub?
+
+:::
```

Comment:
```suggestion

Prepare to summarise and share with the rest of the workshop.
:::
```
I've gone with a call to action to summarise and report afterwards in response to [this comment.](https://github.com/czi-catalystproject/hub-champion-training/pull/23/files#r1481169559)